images/python/3.8.Dockerfile

```diff
@@ -26,7 +26,7 @@ ENV TMPDIR=/tmp \
     BASH_ENV=/home/.bashrc
 
 RUN apk add --no-cache --virtual .build-deps \
-      build-base \
+      build-base tar \
```

Comment:
This will need to be added in a separate apk add as in here it will be removed once built.